.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
```

Comment:
Can we include a comment at the top saying which files this should be checking (similar to [the other repos](https://github.com/alphagov/digitalmarketplace-scripts/blob/master/.dependabot/config.yml))?

Reply:
Ah yeah, good spot

Reply:
Done